File: Wrap-Up/java/PerfectNumber.java
public class PerfectNumber {

    public static void main(String[] args) {
        int number = 28;
        int sumOfDivisors = 0;

       
        for (int count = 1; count < number; count++) {
            if (number % count == 0) {
                sumOfDivisors = sumOfDivisors + count;
            }
        }


        if (sumOfDivisors == number) {
            System.out.println(number + " is a perfect number.");
        } else {
            System.out.println(number + " is not a perfect number.");
        }
    }
}

File: Wrap-Up/javaScript/perfectNumber.js
let number = 28;
let sumOfDivisors = 0;

for (let count = 1; count < number; count++) {
    if (number % count == 0) {
        sumOfDivisors = sumOfDivisors + count;
            }
        }


    if (sumOfDivisors == number) {
            console.log(number + " is a perfect number.");
        } else {
            console.log(number + " is not a perfect number.");
        }

